openmdao/core/problem.py

```diff
@@ -451,7 +455,7 @@ def final_setup(self):
         if Problem._post_setup_func is not None:
             Problem._post_setup_func(self)
 
-    def check_partials(self, out_stream=None, comps=None, compact_print=False,
+    def check_partials(self, out_stream=_DEFAULT_OUT_STREAM, comps=None, compact_print=False,
```

Comment:
looks like `out_stream` and `suppress_output` are redundant?

Reply:
We could change the API and teach users to specify None for outstream instead of using the suppress_output flag. My recommendation is to reject Herb's story and kick this back to him to finish. This is an emergency fix for me.